src/listr/tasks/startNodeTaskFactory.js

```diff
@@ -46,6 +52,10 @@ function startNodeTaskFactory(dockerCompose) {
         enabled: () => isUpdate === true,
         task: async () => dockerCompose.pull(config.toEnvs()),
       },
+      {
+        title: 'Render service templates',
```

Comment:
I feel templates needs for other commands as well (e.g. `generateToAddress`), so probably we should build them in BaseCommand after we determine the current config.

Reply:
This turned out to be the bug that had me stuck for hours trying to figure out why docker was writing dashd.conf as a dir owned by root (because it hadn't been rendered yet). Good catch!